common/utils_unittest.cpp

```diff
@@ -112,25 +112,6 @@ UTILS_TEST(guessResolutionBasic) {
     EXPECT_EQ(h, 0);
 }
 
-UTILS_TEST(guessResolutionOverflow) {
-    std::stringstream sstream;
-    int w, h;
-
-    w = h = std::numeric_limits<int>::max();
-
-    // sanity check
-    ASSERT_GT(std::numeric_limits<long>::max(), w);
-    ASSERT_GT(std::numeric_limits<long>::max(), h);
-
-    sstream << std::numeric_limits<long>::max()
-            << "x"
-            << std::numeric_limits<long>::max();
-
-    EXPECT_FALSE(guessResolution(sstream.str().c_str(), w, h));
-    EXPECT_EQ(w, 0);
-    EXPECT_EQ(h, 0);
-}
-
```

Comment:
I disagree, we should keep this test and fix `guessResolution`.  The `guessResolution` function can take any arbitrary `char *` string that is parsed unconditionally.  If the user specifies a filename that includes dimensions larger than what an `int` can hold, then an overflow occurs.  I agree that in the real world the video resolution won't be that large, but we have no control over how the user names their files (i.e. it could be a misnamed filename).  The `guessResolution` function should validate and handle this case more gracefully.